smf/sled-agent/non-gimlet/config.toml

```diff
@@ -18,18 +18,23 @@ sidecar_revision.soft = { front_port_count = 1, rear_port_count = 1 }
 # in-sync, rather than querying its NTP zone.
 skip_timesync = true
 
-# A file-backed zpool can be manually created with the following:
+# For testing purposes, A file-backed zpool can be manually created with the
+# following:
+#
 # # truncate -s 10GB testpool.vdev
 # # zpool create oxp_d462a7f7-b628-40fe-80ff-4e4189e2d62b "$PWD/testpool.vdev"
 #
 # Note that you'll need to create one such zpool for each below, with a
-# different vdev for each.
+# different vdev for each.  The `create_virtual_hardware.sh` script does this
+# for you.
 zpools = [
   "oxi_a462a7f7-b628-40fe-80ff-4e4189e2d62b",
   "oxi_b462a7f7-b628-40fe-80ff-4e4189e2d62b",
   "oxp_d462a7f7-b628-40fe-80ff-4e4189e2d62b",
   "oxp_e4b4dc87-ab46-49fb-a4b4-d361ae214c03",
   "oxp_f4b4dc87-ab46-49fb-a4b4-d361ae214c03",
+  "oxp_14b4dc87-ab46-49fb-a4b4-d361ae214c03",
+  "oxp_24b4dc87-ab46-49fb-a4b4-d361ae214c03",
```

Comment:
We need two more pools in the system in order to deploy a 5-node CockroachDB cluster.